### DISCUSSION TITLE
Current Contributor License Agreement

### DISCUSSION THREAD

**Discussion by jonathan-albrecht-ibm** [orientechnologies/orientdb#10183] [General] [answered]
Is there currently a Contributor License Agreement for OrientDB? The docs refer to this link:

https://www.clahub.com/agreements/orientechnologies/orientdb

but its no longer working.

**Comment by tglman** [comment]
probably an outdated link, where did you find that ?

**Reply by jonathan-albrecht-ibm** [reply]
I saw it in these two places:

- https://github.com/orientechnologies/orientdb/blob/develop/.github/CONTRIBUTING.md
- https://orientdb.org/docs/3.2.x/misc/Contribute-to-OrientDB.html

Is there currently a Contributor License Agreement or similar that is required when submitting a PR?

**Reply by jonathan-albrecht-ibm** [reply]
Hi @tglman, I just wanted to follow up and check if there is currently any Contributor License Agreement that is required when submitting a PR?

**Reply by tglman** [reply, marked as answer]
Hi,

no that is not needed anymore, as far as all changes come under apache 2 licence is good, will remove the outdated links

**Reply by jonathan-albrecht-ibm** [reply]
Great, thanks @tglman!
